### PR TITLE
Make `make_pad_mask` onnx convertible

### PR DESCRIPTION
**Pull request by Masao-Someki**
## What?
This PR is to implement onnx-convertible `make_pad_mask` function.
This PR is an extension of Ken's PR, #4928.

First, I tried to implement this function based on the Wenet implementation, but I found it difficult to adapt to the usage of ESPnet. So I implemented it based on espnet-onnx implementation.

### CODE REVIEW

**Comment by Masao-Someki** [comment]
## ToDO
- [ ] Add tests to check if the output is the same.
- [ ] Add tests to check if the function is traceable.

File: espnet/nets/pytorch_backend/nets_utils.py
Comment (sw005320) [inline espnet/nets/pytorch_backend/nets_utils.py:183]: Sorry, xs must be specified or xs must not be specified?
I want to confirm it.

**Review by sw005320 — APPROVED** [review]
LGTM
After you fix the CI, I’ll merge this PR.

File: espnet/nets/pytorch_backend/nets_utils.py
Comment (Masao-Someki) [reply espnet/nets/pytorch_backend/nets_utils.py:183]: "`xs` must not be specified" is correct.

**Comment by Masao-Someki** [comment]
Sorry, I realized there is a way to run CI locally after I pushed everything...

**Comment by sw005320** [comment]
@Masao-Someki, please fix https://github.com/espnet/espnet/actions/runs/5634779771/job/15265030888?pr=5326#step:9:5757

I'd recommend you finish this test locally. Then, you do not have to wait for the CI test https://github.com/espnet/espnet/blob/master/CONTRIBUTING.md#4-unit-testing

**Comment by sw005320** [comment]
Thanks a lot!

@Masao-Someki, is it ready to be merged?
(I just want to make sure since this PR affects many parts ant I want to double check)

**Comment by Masao-Someki** [comment]
@sw005320 
Sorry for the late comment. It's ready now!

**Comment by sw005320** [comment]
thanks!

**Pull request merged**
